packages/fxa-settings/src/styles/brand-banner.css

```diff
@@ -14,5 +14,5 @@
 }
 
 .brand-messaging {
-  @apply mb-32;
+  @apply mb-32 mobileLandscape:mb-0;
```

Comment:
Do we even need this `brand-messaging` class at all? I see we're only using it in one place. Tailwind docs recommend only abstracting classes for "very small, highly reusable things like buttons and form controls". Is it for searchability?